ansible/inventories/group_vars/all

```diff
@@ -32,11 +32,6 @@ logstash_forwarder:
 # environment
 parasitic: false
 
-denyhosts_allowed:
```

Comment:
This var is still lurking in our internal repo, even though the role that uses it was removed completely.
The `group_vars/all` files of both repos should be cleaned up according to the changes of this pull-request, and if possible we should look into re-organizing the `group_vars/all` files so that as many internal playbooks as possible can be run solely from the internal repo.